presto-raptor/src/main/java/com/facebook/presto/raptor/storage/ReaderAttributes.java

```diff
@@ -25,23 +25,25 @@
 {
     private final DataSize maxMergeDistance;
     private final DataSize maxReadSize;
-    private final DataSize streamBufferSize;
     private final DataSize tinyStripeThreshold;
+    private final DataSize streamBufferSize;
     private final boolean lazyReadSmallRanges;
+    private final boolean zstdJniDecompressionEnabled;
```

Comment:
Rather than duplicating these configs, just put `OrcReaderOptions` as a member?

Reply:
OrcReaderOptions has three params -> Max block size, tinyStripeThreshold and max merge size.

ReaderAttributes does not contain max block size at all.
So now the question is should we add it to Reader Attributes. 

1. This config will only be used for OrcFileRewriter and OrcStorageManager construction.
As I see, Raptor always configures that to be 1 PB for these use cases.
2. For other use cases what should be the config value for this option since its not used at all.